components/org.wso2.carbon.identity.oauth.endpoint/src/main/java/org/wso2/carbon/identity/oauth/endpoint/user/impl/UserInfoJSONResponseBuilder.java

```diff
@@ -280,4 +281,9 @@ protected String returnSubjectClaim(String sub, String tenantDomain, OAuth2Token
         }
         return sub;
     }
+    private String getUsername(String authorizedUser){
+        String tenantDomain = MultitenantUtils.getTenantDomain(authorizedUser);
+        return  authorizedUser.replace("@"+tenantDomain,"");
```

Comment:
Shouldn't we decide the format of the name based on Service Provider configuration?
ie. Decide whether we should append tenant domain or not, append user store domain or not?